Makefile

```
@@ -2,11 +2,18 @@ build:
 	@yarn turbo run build
 build-sequential:
 	@yarn turbo run --concurrency 1 build
+node-deps:
+	@yarn
+	@$(MAKE) build
 python-deps:
-	@python3 -m pip install -r images/plbase/python-requirements.txt --root-user-action=ignore
+	@python3.12 -m venv .venv
+	.venv/bin/python3 -m pip install --upgrade pip
+	.venv/bin/python3 -m pip install -r images/plbase/python-requirements.txt --root-user-action=ignore
```

I had `python3.10` already installed in `.venv`. When we ran `@python3.12 -m venv .venv` above it correctly added `python3.12` to `.venv/bin`, but in the following lines `python3` still executed `python3.10`.
```suggestion
	.venv/bin/python3.12 -m pip install --upgrade pip
	.venv/bin/python3.12 -m pip install -r images/plbase/python-requirements.txt --root-user-action=ignore
```

---

Would it make sense to add a [Makefile variable](https://www.gnu.org/software/make/manual/html_node/Reference.html) for the python version so it's not repeated throughout the file?